Dockerfile

```diff
@@ -10,7 +10,7 @@ WORKDIR /go/src/app
 COPY . .
 
 # CGO_ENABLED=1 is mandatory for building go-sqlite3
-RUN cd cmd/authelia && GOOS=linux GOARCH=amd64 CGO_ENABLED=1 go build -o authelia
+RUN cd cmd/authelia && GOOS=linux GOARCH=amd64 CGO_ENABLED=1 go build -tags netgo -ldflags '-w' -o authelia
```

Comment:
how is this related to enabling multiarch?

Reply:
Tagged netgo to make sure we use Go built-in net package and not the system’s one.
ldflags -w just disables debug letting the file be smaller.